### DISCUSSION TITLE
Why is CurvePoint in read-fonts and not in write-fonts?

### DISCUSSION THREAD

**Discussion by kardwen** [googlefonts/fontations#1297] [General]
I have written `svg2ttf` in Rust: https://github.com/kardwen/verglas

While doing so i found the separation into the crates `read-fonts` and `write-fonts` somewhat arbitrary. This is especially noticeable when trying to create a glyph where I need `CurvePoint` which are in `read-fonts` but are obviously also needed for drawing glyphs: https://github.com/kardwen/verglas/blob/main/src/forge/font/glyph.rs

I am not experienced with fonts so my project has some rough edges but maybe you are interested in how I used crates from `fontations`? I would also greatly value feedback for it.

**Comment by cmyr** [comment]
`write-fonts` sits on top of `read-fonts`, so everything in read-fonts is always available in write-fonts. Often the types used for reading and writing are different, but sometimes we reuse the same types in both places; this is one of those cases.

I agree this is a papercut, and an easy solution here would be to reexport this type in `write_fonts::tables::glyf::simple`.
